git-branchless-smartlog/tests/test_smartlog.rs

```diff
@@ -812,3 +812,95 @@ fn test_exact() -> eyre::Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_show_signature_flag() -> eyre::Result<()> {
+    let git = make_git()?;
+
+    git.init_repo()?;
+
+    // Just create a regular commit without trying to sign it
+    git.commit_file("signed_file", 1)?;
+    git.commit_file("unsigned_file", 2)?;
```

Comment:
[**question**] Are these filenames/comments correct? I don't know much about signing commits, but it seems like something is missing to actually do the signing.